sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.strongarm.inc

```diff
@@ -13,6 +13,13 @@ function dccr_ftr_workshop_strongarm() {
   $strongarm = new stdClass();
   $strongarm->disabled = FALSE; /* Edit this to true to make a default strongarm disabled initially */
   $strongarm->api_version = 1;
+  $strongarm->name = 'date_format_workshop_reminder';
+  $strongarm->value = 'l, F dS, Y \\a\\t G:i';
+  $export['date_format_workshop_reminder'] = $strongarm;
```

Comment:
I need to test this; but I think date formats doesn't get well exported through features